source/user-manual/reference/ossec-conf/index.rst

```diff
@@ -133,3 +135,4 @@ All of the above sections must be located within the top-level ``<ossec_config>`
    wodle-azure-logs
    wodle-agent-key-polling
    verifying-configuration
+   fluent-forward
```

Comment:
This order makes Fluent forwarder appear under the special chapter "Verifying configuration". I think Fluent forwarder shall be above the _wodle_ sections.